roles/apiv2/templates/caliopen-api.yaml.j2

```diff
@@ -35,7 +35,10 @@ ApiConfig:
     host: {{ hostvars['cache1']['backend_ip'] }}:6379
     password: ""      #no password set
     db: 0             #use default db
-  adminUsername: admin  # username on whose behalf notifiers will act. This admin user must have been created before by other means.
+  NotifierConfig:
+    base_url: https://{{ caliopen_domain_name }}
```

Comment:
does `{{ caliopen_domain_name }}` resolve without a trailing slash ?

Reply:
no it's a FQDN not an URL